boot/bootstate20.go

```diff
@@ -341,11 +335,8 @@ func (ks20 *bootState20Kernel) setNext(next snap.PlaceInfo, bootCtx NextBootCont
 		}
 		// kernels are usually loaded directly by the bootloader, for
 		// which we may need to pass additional data to make 'try'
-		// operation more robust - that might be provided by the
-		// RebootBootloader interface
-		if rbi.RebootBootloader, err = ks20.getRebootBootloader(); err != nil {
-			return RebootInfo{RebootRequired: false}, nil, err
-		}
+		// operation more robust
```

Comment:
I don't think the edited comment is a good explanation for the code, it probably needs to mention that this will help to find the bootloader later to find out about the additional data

Reply:
yea you're right, updated the comment